src/esn.calendar.libs/app/app.module.js

```diff
@@ -29,6 +29,7 @@ angular.module('esn.calendar.libs', [
   'esn.widget.helper',
   'linagora.esn.graceperiod',
   'materialAdmin',
+  // 'esn.lodash-wrapper',
```

Comment:
remove it if not needed

Reply:
I've fixed the problem. Thanks for the review